gateway-server/pom.xml

```diff
@@ -135,6 +142,11 @@
             <artifactId>javax.servlet-api</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>com.google.guava</groupId>
```

Comment:
Would prefer to avoid guava dep if possible

Reply:
Why?

Reply:
Guava is a really ugly dependency to pull in. Historically it broke backwards dependencies. It also conflicts with most of the Hadoop components versions of guava. For example Hadoop and Hive both use a VERY old version of Guava. Once a dependency is pulled in, it is very hard to get rid of it. 

We should try our hardest to not include it if possible. Most Guava stuff is in JDK 8 now so don't need the extra dependency.

Reply:
I'm going to get rid of it then...